src/types/BaseField.ts

```diff
@@ -49,14 +51,44 @@ const writableBaseFieldSchema: JSONSchemaType<WritableBaseField> = {
 			enum: Object.values(BaseFieldScope),
 		},
 	},
-	required: ['label', 'description', 'shortCode', 'dataType', 'scope'],
+	required: ['shortCode', 'dataType', 'scope'],
 };
```

Comment:
Since this is a breaking change to this endpoint, we should take this opportunity to disallow unrecognized keys; see #701.
```suggestion
    additionalProperties: false,
};
```